### PR TITLE
Adding new terms to the glossary

### PR DESCRIPTION
**Pull request by bergerhoffer**

### CODE REVIEW

**Comment by bergerhoffer** [comment]
FYI @openshift/team-documentation - adding entries for "image stream" (two words) and "security context constraints" (noting that it is plural `SecurityContextConstraints`)

**Review by jeana-redhat — APPROVED** [review]
LGTM

File: contributing_to_docs/term_glossary.adoc [hunk 1]
```diff
@@ -514,6 +521,15 @@ Kubernetes API object that holds secret data of a certain type.
 
 See link:https://kubernetes.io/docs/concepts/configuration/secret/[Secrets - Kubernetes].
 
+''''
+=== security context constraints
```
Comment (adellape) [inline contributing_to_docs/term_glossary.adoc:525]: Should we mention explicitly the acronym SCC (of which we've got 150+ hits in the repo), or is allowance assumed? I see we've called out some abbreviations already, like CRD.

Comment (bergerhoffer) [reply contributing_to_docs/term_glossary.adoc:525]: Great idea, I'll add it

**Pull request merged**